autoload/lsp/completion.vim

```diff
@@ -248,6 +248,74 @@ export def CompletionReply(lspserver: dict<any>, cItems: any)
   endif
 enddef
 
+# process the completion documentation without LSP server request
+def ShowCompletionDocumentationNoLsp(cItem: any)
```

Comment:
It looks like most of the code in the CompletionResolveReply() function is duplicated here.  Can you refactor the CompletionResolveReply() function to use this function?